docker-images/zac/README.md

```diff
@@ -0,0 +1,10 @@
+
+# build the openziti/zac image
```

Comment:
this is just a file to remove later... i'd just prefer we don't bother adding a file like this we just plan to remove some day.

Reply:
This answers the predictable question, "Which image do I need? Oh, this one's deprecated. Next." I assume we'll delete the directory when we decommission the node integration.